app/views/landing.html

```diff
@@ -32,6 +32,10 @@ <h1 translate>Search for an event</h1>
     </div>
   </div>
 </div>
+<div class="text-center" ng-if="loadingMsg">
+  <h2>{{loadingMsg}}</h2>
+  <i class="fa fa-spinner fa-spin fa-4x"></i>
+</div>
 <div class="container spacing-above-md" ng-if="!searchComplete">
```

Comment:
I think we should hide this when showing the loading indicator.
Could do: `ng-if="!loadingMsg && !searchComplete"`